src/cdtools/datasets/ptycho_2d_dataset.py

```diff
@@ -398,3 +398,47 @@ def downsample(self, factor=2):
                 self.background.unsqueeze(0).unsqueeze(0),
                 factor,
                 divisor_override=1)[0,0]
+
+
+    def crop_2D_translations(self, roi):
```

Comment:
I think "2D" is superfluous, could we change the name to "crop_translations"?

Reply:
That name sounds better; changed the name to 'crop_translations'.